scripts/docker-compose.yml

```diff
@@ -84,6 +84,34 @@ services:
       - -c
       - 'until /usr/local/bin/psql postgres://testuser:knextest@postgres/knex_test -c "SELECT 1"; do sleep 5; done'
 
+  spanner-emulator:
```

Comment:
can you configure integration tests to run on this?

(probably not all of them would be passing, and it's ok to skip the ones that are hard to fix immediately, but the more coverage we can get, the better)